bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,187 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Setup](#setup) \
+   2.1. [Setup Node and Configure](#setup-node-and-configure) \
+   2.2. [Build node from `x` tag](#build-node-from-x-tag) \
+   2.3. [Initialize the chain](#initialize-the-chain) \
+   2.4. [Update the configuration](#update-the-configuration) \
+   2.5. [Retrieve the genesis file](#retrieve-the-genesis-file) \
+   2.6. [Create a Keyring](#create-a-keyring) \
+   2.7. [Get Funds](#get-funds) \
+   2.8. [What is BLS Voting](#what-is-bls-voting) \
+   2.9. [Create BLS Key](#create-bls-key)
+3. [Run Node](#run-node)
+4. [Create Validator](#create-validator)
+5. [Verify your Validator](#verify-your-validator)
+
+## Prerequisites
+
+This system spec has been tested by validators and found to be comfortable:
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
+- 100MBps bidirectional internet connection
+
+You can run Babylon on lower-spec hardware for each component, but you may find that it is not highly performant or prone to crashing.
+## Setup
+
+We need to setup the following before we can classify as a validator. 
+- Setup node and configure
+- Create keyring
+- Get funds
+- Create BLS Key
+- Run node
+
+### Setup Node and Configure
+
+<!-- TODO: check add in the correct tag for the testnet -->
+#### Build node from `x` tag. 
+
+Here we will checkout the tag in which we will use. 
+
+` git checkout x`
+
+#### Initialize the chain
+
+Run  `babylond init --chain-id bbn-test-5` in the terminal 
+
+It should generate all of the necessary files such as `app.toml`, `client.toml`, `genesis.json`
+
+#### Update the configuration
+
+We need to then change the configuration details in the following files. 
+- In `app.toml` under `[btc-network]`change network to `signet` and `iavl-cache-size = 0`
```

Comment:
Why do we need to set the `iavl-cache-size` to `0`? Also, this might be interpreted as `iavl-cache-size` being under `btc-network`.

Reply:
We discovered this on devnet-3 and confirmed that this flag is needed during this round of performance benchmarking. Basically it prevents the memory of the node from becoming too big.
We also need to add `iavl-disable-fastnode=true`.

@KonradStaniec can suggest the exact justification to provide on the doc for this.